### PR TITLE
Restructure and improve documentation

### PR DESCRIPTION
**Pull request by codeliner**
See #79 and #71

### CODE REVIEW

**Comment by codeliner** [comment]
@prolic here it is

File: docs/event_store.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# Prooph Event Store
+
+Prooph Event Store is the central component of this package. If you are familiar with doctrine
+you can compare it with doctrine's EntityManager.
+However, Prooph Event Store is especially designed to add an centralized, event-driven system on top
```
Comment (prolic) [inline docs/event_store.md]: must be: add a centralized, not add an centralized

File: docs/repositories.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+    $this->assertNotSame($user, $fetchedUser);
+
+    $this->assertEquals('John Doe', $fetchedUser->name());
+
+    $this->assertEquals('contact@prooph.de', $fetchedUser->email());
+}
+```
+
+In the first tes case you can see how a aggregate (the user entity in this case) can be added to the repository.
```
Comment (prolic) [inline docs/repositories.md]: must be: how an aggregate, instead of how a aggregate

**Comment by prolic** [comment]
Just 2 spellings found, fix them and I'll merge it tomorrow, or you can merge it yourself after they are fixed.

Great job! :+1:

**Comment by codeliner** [comment]
thanks. I merge it directly. Can't wait to see it on github (and I want to check that links are working)

**Pull request merged**